cmd/fake-files-create/maker/mock_image_test.go

```diff
@@ -0,0 +1,18 @@
+package maker_test
+
+import (
+	"github.com/AnimationMentor/fake-files/cmd/fake-files-create/maker"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("MockImage", func() {
+	It("makes an PNG", func() {
+		img := maker.MakePNG()
+		Expect(img).NotTo(BeEmpty())
+	})
+	It("makes an JPEG", func() {
+		img := maker.MakeJPEG()
+		Expect(img).NotTo(BeEmpty())
+	})
+})
```

Comment:
the tests don't do much yet, but this is growing out of the scope of one PR so these stubs will do for now